docs/quick-starts/framework/wordpress-plugin/README.mdx

```diff
@@ -0,0 +1,238 @@
+---
+slug: /quick-starts/wordpress-plugin
+sidebar_label: WordPress plugin
+sidebar_custom_props:
+  description: Use official WordPress plugin to integrate Logto into your WordPress website.
```

Comment:
Missing the icon file name?
`logoFilename: wordpress.svg`

Reply:
i believe by default it will find `wordpress-plugin.svg`